tests/impl.cpp

```diff
@@ -1991,7 +1991,13 @@ result_t test_mm_store_ps(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_store_ss(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    float *p = (float *) impl.mTestFloatPointer1;
+    float x = impl.mTestFloats[i + 4];
```

Comment:
```
float x = impl.mTestFloats[i];
float p[4];
```

I think we should not store the value into input data.

Reply:
I refered to `test_mm_store_ps()`
`impl.mTestFloats` is taking a random float as testing data

Reply:
You mean referred from `test_mm_store_ps()`?
We still have random float in this modification.

Reply:
Is there any suggested way to get a random testing number

Reply:
We already have it.
The `impl.mTestFloats` array is generated with random float values.

Reply:
so I can just simply use `impl.mTestFloats[0]`?

Reply:
Use `impl.mTestFloats[i]`.

Reply:
got it